doc/NPCs.md

```diff
@@ -228,21 +263,53 @@ The dynamic line is chosen based on whether the player character is performing a
 }
 ```
 
+### Based on the days since the Cataclysm
+The dynamic line is chosen based on the specified number of days have elapsed since the start of the Catacylsm.  Both entries are optional.  The line from `yes` will be shown if the that many days have passed, otherwise the line from `no`.
```

Comment:
The description suggests it must be exactly that many days, but I think the implementation is at least that many days.

Reply:
Fixed.